common/content/services.js

```diff
@@ -16,7 +16,7 @@ const Services = Module("services", {
         this.jsm = window.Services;
         this.services = {
             "autoCompleteSearch": {
-                class_: "@mozilla.org/autocomplete/search;1?name=history",
+                class_: "@mozilla.org/autocomplete/search;1?name=" + (services.get("vc").compare(VERSION, "49") < 0 ? "history" : "unifiedcomplete"),
```

Comment:
Vimperator does not work at all in add-on devel builds after applying this commit as-is.
However, changing `history` to `unifiedcomplete` without the version check works.